WooCommerce/Classes/ViewRelated/Dashboard/Factories/StatsDataTextFormatter.swift

```diff
@@ -100,7 +100,7 @@ struct StatsDataTextFormatter {
 
     // MARK: Views and Visitors Stats
 
-    /// Creates the text to display for the visitor count.
+    /// Creates the text to display for the visitor count based on SiteVisitStats data and a given interval.
     ///
     static func createVisitorCountText(siteStats: SiteVisitStats?, selectedIntervalIndex: Int?) -> String {
```

Comment:
Can we make `selectedIntervalIndex` non-optional here?
```suggestion
    static func createVisitorCountText(siteStats: SiteVisitStats?, selectedIntervalIndex: Int) -> String {
```

Reply:
Yes, I overlooked that we could do that now! 👍